apps/video/test/marionette/lib/video.js

```diff
@@ -130,14 +133,30 @@ Video.prototype = {
    * @return {Marionette.Element} First element of all thumbnail images.
    */
   get thumbnail() {
-    return this.client.findElement(Video.Selector.thumbnail);
+    return this.client.helper.waitForElement(Video.Selector.thumbnail);
   },
 
   /**
    * @return {Marionette.Element} video player element.
    */
   get player() {
-    return this.client.findElement(Video.Selector.player);
+    return this.client.helper.waitForElement(Video.Selector.player);
+  },
+
+  showControls: function() {
+    this.player.tap();
```

Comment:
Okay, reading the docs, it looks like you can do something like
```
var elapsed = this.client.findElement(Video.Selector.elapsedTime)
if (!elapsed.displayed()) {
   this.player.tap();
   this.client.waitFor(function() { return elapsed.displayed(); }
}
```

I do not understand how marionette works, so I'm not confident that that function does what I think it does, though.